robyn/__init__.py

```diff
@@ -103,15 +103,17 @@ def startup_handler(self, handler: Callable) -> None:
     def shutdown_handler(self, handler: Callable) -> None:
         self._add_event_handler(Events.SHUTDOWN, handler)
 
-    def start(self, url: str = "127.0.0.1", port: int = 5000):
+    def start(self, url: str | None = None, port: int | None = None):
```

Comment:
can you make the add the default parameters like the original and then instead of 
```
        if url is None:
            url = os.getenv("ROBYN_URL", "127.0.0.1")
        if port is None:
            port = int(os.getenv("ROBYN_PORT", "5000"))
```

we can do 


```
  url = os.getenv("ROBYN_URL", url)
  port = int(os.getenv("ROBYN_PORT", port))
```

Reply:
Also, you will have to import `from typing import Union` instead of `|` as we are supporting python versions before Python 3.10 and `|` was introduced in 3.10/

Reply:
I think it is better changing the code to
```

url = os.getenv("ROBYN_URL", url)
  port = int(os.getenv("ROBYN_PORT", port))

```

And remain the default parameters like the original.